sources/deny.toml

```diff
@@ -52,8 +52,11 @@ skip = [
     # older version used by hyper 0.14.2
     { name = "socket2", version = "0.3.19" },
 
-    # older version used by tough 0.11.2
-    { name = "serde_plain", version = "0.3.0" },
+    # older version used by webpki-roots 0.21.1
```

Comment:
I looked at the lock file and I don't see a change to the webpki-roots, which is a good thing because of https://github.com/bottlerocket-os/bottlerocket/blob/5a083d917e0c1101bec5a8668f5a5a8048239454/sources/webpki-roots-shim/Cargo.toml#L3..L11

Reply:
Yeah. In this case, one of our other deps (I believe `actix-web`) moved forward to use a newer version of `pem`. Our cargo-deny config flags differences in dependency versions, but we can't change webpki-roots due to what you've linked.